Plugins/Creature/Creature.cpp

```diff
@@ -2397,53 +2438,66 @@ void Creature::InitCriticalRangeHook()
                 int32_t retVal;
                 if (!bOffHand) //mainhand
                 {
-                    if (auto critRngOvr = g_plugin->GetServices()->m_perObjectStorage->Get<int32_t>(pThis->m_pBaseCreature->m_idSelf, "CRITICAL_RANGE_OVERRIDE!1"))
-                    {
+                    auto pItem = pThis->m_pBaseCreature->m_pInventory->GetItemInSlot(Constants::EquipmentSlot::RightHand);
+                    std::string nBaseItemID;
```

Comment:
Ideally don't use hungarian notation prefixes at all, but please don't use `n` for strings.

Reply:
Removed all Hungarian (Including some that was from the previous incarnation)